src/Components/Create.tsx

```diff
@@ -0,0 +1,163 @@
+import React, { useState } from "react";
+import { PlusOutlined } from "@ant-design/icons";
+import { useNavigate } from "react-router-dom";
+import { Form, Input, DatePicker, InputNumber, Upload, notification, Rate } from "antd";
+import { ButtonDiv, StandardButton } from "./Style";
+
+
+const { RangePicker } = DatePicker;
+const { TextArea } = Input;
+
+type DateSetter = {
+    start: Date | undefined,
+    end: Date | undefined
+}
+
+const desc = ['terrible', 'bad', 'normal', 'good', 'wonderful'];
```

Comment:
Egy ratingOptions elnevezes talan leirobb lenne. Ha nagyon overengineeringelni akarunk itt szoba johet egy enum is esetleg. Pilu, ha van ido beszeljetek az enumokrol esetleg.